contracts/WildcardSteward_v3.sol

```diff
@@ -904,10 +875,10 @@ contract WildcardSteward_v3 is Initializable {
             artistAmount = totalAmount.mul(artistPercentages[tokenId]).div(
                 10000
             );
+            deposit[artistAddresses[tokenId]] = deposit[artistAddresses[tokenId]]
+                .add(artistAmount);
```

Comment:
This is for gas efficiency. no point in adding to artists deposit if artistAmount =0.